test/libsolidity/semanticTests/libraries/library_call_in_homestead.sol

```diff
@@ -0,0 +1,13 @@
+library Lib { function m() public returns (address) { return msg.sender; } }
```

Comment:
Interesting that this case does not need to set EVM version even though the name suggests that it should. Maybe it's testing stuff that would not work before homestead rather than after it?

Reply:
This tests a new feature of homestead: delegatecall. We should restrict the evm version to `>= homestead`.

Reply:
Added  `EVMVersion: >=homestead` to the test case file